src/Compiler/Driver/CreateILModule.fs

```diff
@@ -584,19 +584,22 @@ module MainModuleBuilder =
                 [ resource ]
 
         // a user cannot specify both win32res and win32manifest
-        if not (tcConfig.win32manifest = "") && not (tcConfig.win32res = "") then
+        if
+            not (tcConfig.win32manifest.Length = 0)
```

Comment:
Shouldn't this also then be compared via IsNullOrEmpty?

Reply:
Also applies to a few other places in the PR where x.Length is used. 

The rules says that 
> Testing for null does not significantly affect the relative performance

hence it's probably worth just putting it everywhere like this as opposed to risking any NREs in the compiler code...

Reply:
One could argue that it's not strictly needed in these cases but yeah, let's be on the safe side and consistent :)

Reply:
Yeah it's also for cases when someone later decides to copypaste this code to a less safe context or add some logic in between... You know, paranoid mood today.